assets/aircraft/BE9T.json

```
@@ -0,0 +1,37 @@
+{
+    "name": "Beech King Air",

```

```suggestion
    "name": "Beech King Air 90",
```
or
```suggestion
    "name": "Beech 90 King Air",
```

I'd suggest adding the "90" to the name, since Beech King Air is a whole aircraft family.